[IMP] stock_manual_transfer: rename group's external ID T#79454

luisg123v opened the pull request:
Rename group's external ID, from:
`group_stock_manual_transferp` → `group_user`
to follow usual naming conventions.

The reason for the old name is first implementation of this was not as a standalone module.

In addition, IDs and names of model access rights are also modified to
follow conventions.

CLaurelB commented on stock_manual_transfer/security/ir.model.access.csv:
```diff
@@ -1,3 +1,3 @@
 id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
-access_stock_can_stock_manual_transfer,stock_can_stock_manual_transfer,model_stock_manual_transfer,group_stock_manual_transfer,1,1,1,1
```
Does changing the ID and name of the access right require updating the records in the ir_model_data and ir_model_access tables?

Additionally, it seems this change is missing from the commit description.

luisg123v replied:
@CLaurelB 

> Does changing the ID and name of the access right require updating the records in the ir_model_data and ir_model_access tables?

No, because these are updatable records (`noupdate=0`). It would be similar to update views.

> Additionally, it seems this change is missing from the commit description.

I didn't include it because I didn't consider it as necessary, but done, "better explicit than implicit".

CLaurelB commented:
LGTM 👍

luisg123v committed:
[IMP] stock_manual_transfer: rename group's external ID

Rename group's external ID, from:
`group_stock_manual_transferp` → `group_user`
to follow usual naming conventions.

The reason for the old name is first implementation of this was not as
a standalone module.

In addition, IDs and names of model access rights are also modified to
follow conventions.

TeposteAJ commented:
@luisg123v  LGTM 👍

ajteposte reviewed (approved):
@luisg123v LGTM 👍
